loony_dev/web/static/ds/ld-system.css

```
@@ -0,0 +1,353 @@
+  --rail-w: 236px;
+  --side-w: 300px;
+  --pad-card: 18px;
+  --row-gap: 12px;
+}
+.dense { --pad-card: 13px; --row-gap: 8px; }
+
+.material-symbols-outlined {
+  font-family: 'Material Symbols Outlined';
+  font-weight: normal; font-style: normal;
+  line-height: 1; letter-spacing: normal; text-transform: none;
+  display: inline-block; white-space: nowrap; word-wrap: normal; direction: ltr;
```

Stylelint (17.13.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)